SWLOR.Game.Server/Feature/GuiDefinition/MusicManagerDefinition.cs

```diff
@@ -0,0 +1,105 @@
+﻿using SWLOR.Game.Server.Feature.GuiDefinition.ViewModel;
+using SWLOR.Game.Server.Service.GuiService;
+
+namespace SWLOR.Game.Server.Feature.GuiDefinition
+{
+    public class MusicManagerDefinition : IGuiWindowDefinition
+    {
+        private readonly GuiWindowBuilder<MusicManagerViewModel> _builder = new();
+
+        public GuiConstructedWindow BuildWindow()
+        {
+            _builder.CreateWindow(GuiWindowType.MusicManager)
+                .SetIsResizable(true)
+                .SetIsCollapsible(true)
+                .SetInitialGeometry(0, 0, 475f, 650f)
+                .SetTitle("Music Manager")
+
+                // Main Search - Top Window Row
+                .AddColumn(colSearch =>
+                {
+                    colSearch.AddRow(row =>
+                    {
+                        row.AddTextEdit()
+                            .SetPlaceholder("Search for Music")
+                            .BindValue(model => model.SearchText);
+
+                        row.AddButton()
+                            .SetText("X")
+                            .SetHeight(35f)
+                            .SetWidth(35f)
+                            .BindOnClicked(model => model.OnClickClearSearch());
+
+                        row.AddButton()
+                            .SetText("Search")
+                            .SetHeight(35f)
+                            .BindOnClicked(model => model.OnClickSearch());
+                    });
+
+                    // Main Window Container - Second Window Row
+                    colSearch.AddRow(row =>
+                    {
+                        row.AddColumn(colMusic =>
+                        {
+                            colMusic.SetHeight(500f);
```

Comment:
The window does not resize gracefully and I suspect it's because you've hard coded the height of this control to 500. Is there a reason for this? Can you remove it?